Lib/fontTools/otlLib/builder.py

```diff
@@ -3,6 +3,9 @@
 from fontTools import ttLib
 from fontTools.ttLib.tables import otTables as ot
 from fontTools.ttLib.tables.otBase import ValueRecord, valueRecordFormatDict
+from fontTools.ttLib.tables import otBase, otTables
+from fontTools.otlLib.error import OTLLibError
+from fontTools.otlLib.classDefBuilder import ClassDefBuilder
```

Comment:
why the proliferation of submodules? can't ClassDefBuilder live inside otlLib.builder? It's only one class.

Reply:
Why *not* a proliferation of submodules? ClassDefBuilder is currently buried in the middle of a large and very mixed collection of functions. It's a sufficiently self-contained bit of functionality that moving it to its own file helps with code organisation.